CONTRIBUTING.md

```diff
@@ -0,0 +1,236 @@
+If you want to know all the issues we're dealing with right now, take a look to our [glpi-plugin Issue Dashboard](../issues) and look for areas in which you can help.
+
+### :fire_engine:
+### 1.2 Pull Request Dashboard
+If you want to give us a hand solving issues then great, take a look to our [glpi-plugin Pull Request Dashboard](../pulls) and check for an open or closed PR. We don’t want to duplicate efforts.
+
+# 2. Assistance
+
+### :loudspeaker:
+## 2.1 Live Support
+You can find us in [Telegram](https://t.me/flyvemdm), we'll help you as soon as possible.
+
+### :closed_book:
+## 2.2 Technical Questions
+For general technical questions, post an appropriately tagged question on [StackOverflow](http://stackoverflow.com/).
+
+### :speech_balloon:
+## 2.3 Discussion
+For general discussion, use the [Flyve-MDM mailing list](http://mail.ow2.org/wws/info/flyve-mdm-dev).
+
+### :ear:
+## 2.4 Customers Assistance
+Use our official [support channel](https://support.teclib.com/).
+
+# 3. Feature Requests
+
+## 3.1 Requirement for a Feature Request
+
+### 3.1.1 Major Feature Request
+For a major new feature request, [open an Issue](../issues/new) and outline your proposal so it can be discussed.
+
+### 3.1.2 Minor Feature Request
+For a minor new feature request, you can craft it and directly submit it as a Pull Request, we'll take care of it.
+
+## 3.2 _Request a New_ Feature
+You can request a new feature by [submitting an Issue](../issues/new)
+
+## 3.3 _Implement a New_ Feature
+If you like to _implement_ a new feature please [submit an Issue](../issues/new) with a proposal, so we can be sure it's relevant.
+
+# 4. Submitting
+
+## 4.1 How to Submit an Issue.
+
+### 4.1.1 Check for Past Issues.
+Before submitting the issue please check the [Issue Tracker](https://github.com/flyve-mdm//issues), maybe the bug was already reported by another contributor. By doing this you help us maximize the effort spent on solving problems and the addition of new features.
+
+### 4.1.2 Information Needed.
+We require the following information:
+
+* :warning: **Observed Results:** A brief description of the problem.
+* :boom: **Expected Results:** What did you expect to happen?
+* :mag: **Relevant Code:** Code that you find relevant with this issue. You can use [Gist](https://gist.github.com/).
+* :computer: **Context:** Provide context, project version, OS version, device model, etc.
+* :bulb: **Suggest a Fix:** Point out what might be causing the problem.
+
+### 4.1.3 :rocket: Submit an Issue.
+Having all data and hand, file the new issue by filling out our [Issue form](../issues/new).
+### That's it! :tada:
+
+## 4.2 How to Create a Pull Request (PR).
+
+Before submitting your Pull Request check for an open or closed PR that relates to your submission. We don't want to duplicate efforts.
+
+### 4.2.1 Create a Branch (PR)
+
+Create a new branch before committing any changes. Name it anything except _master, develop, release-*, or hotfix-*_.
```

Comment:
git flow requires the branch has a the **feature/** prefix in its name

Reply:
We also may consider using the extended git flow model. I don't like to prefix **feature/* on a bugfix branch. This is controversal, but debian based distros provide a git flow CLI tool with extended features.

Reply:
I would also recomment to adopt a gitlab habit: use the issue ID in  the branch name. THis would result to something like this **feature/111_instalation_wizard**. This is handy to know the issue associated to a branch, if any, to add the **fix # 111** in the commit message, and to autocomplete a branch name.